config/HH_bbWW/Run3_2022EE/samples.yaml

```diff
@@ -0,0 +1,640 @@
+#######GGF########
+GluGlutoHHto2B2V_kl-1p00_kt-1p00_c2-0p00:
+  sampleType: HHnonRes
+  generator: powheg
+  #crossSection: To_be_added
+GluGlutoHHto2B2Vto2L2Nu_kl-1p00_kt-1p00_c2-0p00:
+  sampleType: HHnonRes
+  generator: powheg
+  #crossSection: To_be_added
+GluGlutoHHto2B2Vto2L2Nu_kl-2p45_kt-1p00_c2-0p00:
+  sampleType: HHnonRes
+  generator: powheg
+  #crossSection: To_be_added
+#######VBF########
+VBFHHto2B2V_CV_1_C2V_0_C3_1:
+  sampleType: HHnonRes
+  generator: madgraph
+  #crossSection: To_be_added
+VBFHHto2B2V_CV_1_C2V_1_C3_1:
+  sampleType: HHnonRes
+  generator: madgraph
+  #crossSection: To_be_added
+VBFHHto2B2V_CV-1p74_C2V-1p37_C3-14p4:
+  sampleType: HHnonRes
+  generator: madgraph
+  #crossSection: To_be_added
+VBFHHto2B2V_CV-m0p012_C2V-0p030_C3-10p2:
+  sampleType: HHnonRes
+  generator: madgraph
+  #crossSection: To_be_added
+VBFHHto2B2V_CV-m0p758_C2V-1p44_C3-m19p3:
+  sampleType: HHnonRes
+  generator: madgraph
+  #crossSection: To_be_added
+VBFHHto2B2V_CV-m0p962_C2V-0p959_C3-m1p43:
+  sampleType: HHnonRes
+  generator: madgraph
+  #crossSection: To_be_added
+VBFHHto2B2V_CV-m1p21_C2V-1p94_C3-m0p94:
+  sampleType: HHnonRes
+  generator: madgraph
+  #crossSection: To_be_added
+VBFHHto2B2V_CV-m1p60_C2V-2p72_C3-m1p36:
+  sampleType: HHnonRes
+  generator: madgraph
+  #crossSection: To_be_added
+VBFHHto2B2V_CV-m2p12_C2V-3p87_C3-m5p96:
+  sampleType: HHnonRes
+  generator: madgraph
+  #crossSection: To_be_added
+VBFHHto2B2Vto2L2Nu_CV_1_C2V_0_C3_1:
+  sampleType: HHnonRes
+  generator: madgraph
+  #crossSection: To_be_added
+VBFHHto2B2Vto2L2Nu_CV_1_C2V_1_C3_1:
+  sampleType: HHnonRes
+  generator: madgraph
+  #crossSection: To_be_added
+VBFHHto2B2Vto2L2Nu_CV-1p74_C2V-1p37_C3-14p4:
+  sampleType: HHnonRes
+  generator: madgraph
+  #crossSection: To_be_added
+VBFHHto2B2Vto2L2Nu_CV-m0p012_C2V-0p030_C3-10p2:
+  sampleType: HHnonRes
+  generator: madgraph
+  #crossSection: To_be_added
+VBFHHto2B2Vto2L2Nu_CV-m0p758_C2V-1p44_C3-m19p3:
+  sampleType: HHnonRes
+  generator: madgraph
+  #crossSection: To_be_added
+VBFHHto2B2Vto2L2Nu_CV-m0p962_C2V-0p959_C3-m1p43:
+  sampleType: HHnonRes
+  generator: madgraph
+  #crossSection: To_be_added
+VBFHHto2B2Vto2L2Nu_CV-m1p21_C2V-1p94_C3-m0p94:
+  sampleType: HHnonRes
+  generator: madgraph
+  #crossSection: To_be_added
+VBFHHto2B2Vto2L2Nu_CV-m1p60_C2V-2p72_C3-m1p36:
+  sampleType: HHnonRes
+  generator: madgraph
+  #crossSection: To_be_added
+VBFHHto2B2Vto2L2Nu_CV-m2p12_C2V-3p87_C3-m5p96:
+  sampleType: HHnonRes
+  generator: madgraph
+  #crossSection: To_be_added
+############## Finished non -Resonant Signal samples)##############
+############## Data ##############
+EGamma_Run2022E:
+  sampleType: data
+EGamma_Run2022F:
+  sampleType: data
+EGamma_Run2022G:
+  sampleType: data
+Muon_Run2022E:
+  sampleType: data
+Muon_Run2022F:
+  sampleType: data
+Muon_Run2022G:
+  sampleType: data
+MuonEG_Run2022E:
+  sampleType: data
+MuonEG_Run2022F:
+  sampleType: data
+MuonEG_Run2022G:
+  sampleType: data
```

Comment:
- data datasets should be defined in `config/Run3_2022EE/samples.yaml`
- define all datasets that we can potentially use in bbWW or bbtautau analyses